src/scripts/underground/Underground.ts

```diff
@@ -6,7 +6,7 @@ class Underground implements Feature {
 
     upgradeList: Array<Upgrade>;
     defaults: Record<string, any>;
-    private _energy: KnockoutObservable<number> = ko.observable(0);
+    private _energy: KnockoutObservable<number> = ko.observable(50);
```

Comment:
Could you use `Underground.BASE_ENERGY_MAX` here instead of magic numbers please.